### ISSUE TITLE
ZIO 2.0: Use consistent naming convention in Gen.*

### ISSUE THREAD

**Issue by jdegoes** [zio/zio#5441]
Currently we have a mix of naming conventions for Gen constructors in the Gen companion object, e.g. `option` versus `anyInt`. I feel we should standardize on the shorter naming convention (because if you standardize on the longer one, every method name ends up prefixed with `any`.

**Comment by adamgfraser** [comment]
The issue here is that we do have `int` to return random integers within a specified range. Maybe we can rename those variants to something like `intBounded`?

**Comment by jdegoes** [comment]
How about `val int` versus `def int(...)`?

**Comment by adamgfraser** [comment]
Just overload it?

**Comment by jdegoes** [comment]
Yes, just overload it (I don't think Scala has problems with this). If this isn't an option, then I would agree `intBounded` or `intRanged` (suffix-style) is better / more IDE friendly, and more consistent with ZIO.

**Comment by adamgfraser** [comment]
Yeah I think that works. If we did want to give it a separate name `intBetween` might be more consistent with our existing naming of the operators on `Random`.

**Comment by jdegoes** [comment]
Either way is good by me. 👍
